pkg/exporter/kafka_proto.go

```diff
@@ -28,6 +29,26 @@ func (kp *KafkaProto) ExportFlows(input <-chan []*flow.Record) {
 	}
 }
 
+func getIPKey(ip [16]uint8) string {
```

Comment:
nit: should we move this util to record.go where we keep other tools for conversion ?
also I didn't take a close look at net package to see if there is another ways to doing this but I assume u checked ?

Reply:
I realized that are transforming from byte array to string and then back to byte array.

I reworked the PR to directly work with byte arrays.